### PR TITLE
[AIFlow] Improve overview.md document.

### PR DESCRIPTION
**Pull request by wuchaochen**
## What is the purpose of the change

[AIFlow] Improve overview.md document.

### CODE REVIEW

File: docs/content/architecture/overview.md [hunk 1]
```diff
@@ -38,40 +38,44 @@ AI Flow consists of four parts: SDK, Meta Service, Notification Service and Sche
       
       * Control Edge: It describes the event trigger conditions for controlling Job.
       
-    * __Translator__: The role of the Translator is to convert the AI Graph defined by the AI Flow program 
+    * __Translator__: The role of the Translator is to convert the AI Graph defined by the AIFlow program 
       into a Workflow.
    
-2. __Meta Service__: Meta service is responsible for managing meta information of AI Flow project. 
-   It includes dataset meta, model meta, job meta, artifact meta, metric meta and project meta, etc.
+2. __AIFlow Server__: AIFlow Server provides meta information management and scheduling services.
```
Comment (jiangxin369) [inline docs/content/architecture/overview.md:44]: I think model center should also be metioned.

File: docs/content/architecture/overview.md [hunk 2]
```diff
@@ -169,31 +190,20 @@ The functions of the Notification Service in the AI Flow system are as follows:
...
 
-The following figure shows the principle of Scheduler Service:
-
-![Alt text](../images/flink-ai-flow/design/scheduler_service.jpg)
-
-1. The user submits Workflow to the Scheduler Service.
-2. The Scheduler Service forwards user requests to the scheduler based on event scheduling.
-3. The scheduler manages the behavior of Workflow, such as running jobs, stopping jobs, etc.
+Scheduler: Scheduler is responsible for executing workflow and must support event-based scheduling.
+Currently, we have implemented an [event-based scheduler](https://github.com/flink-extended/ai-flow/tree/master/lib/airflow) by combining AirFlow and Notification Server.
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:202]:
```suggestion
At present, the default scheduler is the event based scheduling version of Airflow which is implemented with the Notification Service.
```

File: docs/content/architecture/overview.md [hunk 3]
```diff
@@ -144,11 +150,26 @@ The services provided are shown in the figure below:
...
+1. The user submits Workflow to the Scheduler Service.
+2. The Scheduler Service forwards user requests to the scheduler based on event scheduling.
+3. The scheduler manages the behavior of Workflow, such as running jobs, stopping jobs, etc.
 
-## Notification Service
+## Notification Server
 
-Notification Service: It is a service that provides event sending and event notification.
-The functions of the Notification Service in the AI Flow system are as follows:
+Notification Server: It is a service that provides event sending and event notification.
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:171]:
```suggestion
Notification Server: It provides notification service which support the event publishing and subscription.
```

**Review by SteNicholas — CHANGES_REQUESTED** [review]

File: docs/content/architecture/overview.md [hunk 4]
```diff
@@ -38,40 +38,44 @@ AI Flow consists of four parts: SDK, Meta Service, Notification Service and Sche
...
+3. __Notification Server__: Notification Server provides notification service which support the event publishing and subscription.
 
-4. __Scheduler Service__: The functions of the Scheduler Service include submitting workflow, 
-   running workflow, stopping workflow, etc.
+4. __Scheduler__: Scheduler provides the function of executing workflow.
 
 ## Principle
 
-The figure below shows the principle of AI Flow:
+The figure below shows the principle of AIFlow:
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:57]:
```suggestion
The principle of AIFlow is shown in the figure:
```

File: docs/content/architecture/overview.md [hunk 5]
```diff
@@ -88,8 +92,7 @@ AI Flow API mainly includes 4 categories:
 
 AI Graph: As shown in the figure below, it consists of AI Node, Data Edge and Control Edge .
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:93]:
```suggestion
AI Graph: As shown in the figure below, the AI Graph is consisted of the AI Node, Data Edge and Control Edge . 
```

File: docs/content/architecture/overview.md [hunk 6]
```diff
@@ -115,40 +118,56 @@ Jobs and Control Edges make up a Workflow.
 
 Translator: It converts the AI Graph into the Workflow that the scheduler can execute.
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:119]:
```suggestion
Translator: It translates the AI Graph to the executable workflow for Scheduler.
```

File: docs/content/architecture/overview.md [hunk 7]
```diff
@@ -115,40 +118,56 @@ Jobs and Control Edges make up a Workflow.
...
+![Alt text](../images/architecture/translator.png)
 
 The working steps of the translator are as follows:
 1. A group of AI Nodes with the same job config are combined into an AISubGraph.
 2. Call the corresponding JobGenerator according to the job config to convert all AISubGraphs to the corresponding Jobs.
 3. Add all Control Edges to the Workflow.
 
-## Meta Service
+## AIFlow Server
+AIFlow Server provides meta information management and scheduling services.
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:129]:
```suggestion
AIFlow Server supports the Meta Service and Scheduling Service to provide the basic service of the AIFlow.
```

File: docs/content/architecture/overview.md [hunk 8]
```diff
@@ -169,32 +188,22 @@ The functions of the Notification Service in the AI Flow system are as follows:
...
+## Scheduler
 
-The following figure shows the principle of Scheduler Service:
-
-![Alt text](../images/flink-ai-flow/design/scheduler_service.jpg)
-
-1. The user submits Workflow to the Scheduler Service.
-2. The Scheduler Service forwards user requests to the scheduler based on event scheduling.
-3. The scheduler manages the behavior of Workflow, such as running jobs, stopping jobs, etc.
+Scheduler: Scheduler is responsible for workflow&job execution and support event-based scheduling.
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:199]:
```suggestion
Scheduler: Scheduler is responsible for the execution of the workflow and job with the event-based scheduling mechanism.
```

File: docs/content/architecture/overview.md [hunk 9]
```diff
@@ -115,40 +118,56 @@ Jobs and Control Edges make up a Workflow.
...
+1. The user submits Workflow to the Scheduler Service.
+2. The Scheduler Service forwards user requests to the scheduler based on event scheduling.
+3. The scheduler manages the behavior of Workflow, such as running jobs, stopping jobs, etc.
 
-## Notification Service
+## Notification Server
 
-Notification Service: It is a service that provides event sending and event notification.
-The functions of the Notification Service in the AI Flow system are as follows:
+Notification Server: It provides notification service which support the event publishing and subscription.
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:169]:
```suggestion
Notification Server: It provides Notification Service that supports the event publishing and subscription.
```

File: docs/content/architecture/overview.md [hunk 10]
```diff
@@ -115,40 +118,56 @@ Jobs and Control Edges make up a Workflow.
...
+### Meta Service
 
 Meta Service: Provides the CRUD(Create, Read, Update and Delete) service of metadata generated in the workflow of machine learning tasks.
 
 The services provided are shown in the figure below:
 
-![Alt text](../images/flink-ai-flow/design/meta.jpg)
+![Alt text](../images/architecture/meta.png)
 
 1. Dataset: It provides dataset meta-information services, including data address, data format and other information.
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:139]: All `meta-information` could update to `meta`.

File: docs/content/architecture/overview.md [hunk 11]
```diff
@@ -115,40 +118,56 @@ Jobs and Control Edges make up a Workflow.
...
+4. Model: It provides Models meta-information services.
    It contains the description information of the model and the version information of the model.
-5. Metric: It provides metric meta-information services, such as model version evaluate metric, etc.
-6. Artifacts: It provides artifacts meta-information services such as configuration files, jar packages etc.
+5. Metric: It provides metric meta-information services, such as model version evaluate metric.
+6. Artifact: It provides artifacts meta-information services such as configuration files, jar packages etc.
+
+### Scheduling Service
+
+Scheduling Service: The Scheduling Service converts the workflow into an object that can be recognized by the scheduler
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:153]:
```suggestion
Scheduling Service: The Scheduling Service converts the workflow translated into the executable workflow recognized by Scheduler and schedules the workflow execution.
```

File: docs/content/architecture/overview.md [hunk 12]
```diff
@@ -115,40 +118,56 @@ Jobs and Control Edges make up a Workflow.
...
-5. Metric: It provides metric meta-information services, such as model version evaluate metric, etc.
-6. Artifacts: It provides artifacts meta-information services such as configuration files, jar packages etc.
+5. Metric: It provides metric meta-information services, such as model version evaluate metric.
+6. Artifact: It provides artifacts meta-information services such as configuration files, jar packages etc.
+
+### Scheduling Service
+
+Scheduling Service: The Scheduling Service converts the workflow into an object that can be recognized by the scheduler 
+and submits it to the scheduler.
+The traditional scheduler can only schedule batch jobs (when upstream job finished then downstream jobs can run)
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:155]:
```suggestion
The traditional scheduler could only supports the scheduling of the batch jobs(It means that after upstream job finished, downstream jobs could run.)
```

File: docs/content/architecture/overview.md [hunk 13]
```diff
@@ -115,40 +118,56 @@ Jobs and Control Edges make up a Workflow.
...
+5. Metric: It provides metric meta-information services, such as model version evaluate metric.
+6. Artifact: It provides artifacts meta-information services such as configuration files, jar packages etc.
+
+### Scheduling Service
+
+Scheduling Service: The Scheduling Service converts the workflow into an object that can be recognized by the scheduler 
+and submits it to the scheduler.
+The traditional scheduler can only schedule batch jobs (when upstream job finished then downstream jobs can run) 
+but in the online learning scenario where we have jobs that will never finish, it does not meet the demand. 
+The scheduler for Scheduler Service docking must support event-based scheduling.
```
Comment (SteNicholas) [inline docs/content/architecture/overview.md:157]: Add the introduction of the event-based scheduling. The users don't understand what the event-based scheduling is.

**Review by SteNicholas — CHANGES_REQUESTED** [review]

**Review by SteNicholas — APPROVED** [review]

**Pull request merged**